package.json

```diff
@@ -41,6 +41,7 @@
         "static-kdtree": "^1.0.2",
         "sweetalert": "^2.1.2",
         "three": "^0.90.0",
+        "three-css2drender": "^1.0.0",
```

Comment:
Why do we need to add another dependency, when three.js already has CSS2DRenderer: https://github.com/mrdoob/three.js/search?q=CSS2DRenderer&unscoped_q=CSS2DRenderer ?